src/test/with-feature-toggles.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { describe } from 'riteway';
+import dom from 'cheerio';
+import { withFeatureToggles } from '../with-feature-toggles';
+
+import { Consumer } from '../context';
+
+const render = ReactDOMServer.renderToStaticMarkup;
+
+const createTestComponent = () => ({ propCheck } = {}) => (
+  <Consumer>
+    {features => {
+      return (
+        <div>
+          <div className="features">{features.toString()}</div>
+          <div className="prop-check">{propCheck}</div>
+        </div>
+      );
+    }}
+  </Consumer>
+);
+
+describe('withFeatureToggles()', async should => {
+  const { assert } = should();
+
+  {
+    const Component = createTestComponent();
+    const Page = withFeatureToggles()(Component);
+    const $ = dom.load(render(<Page />));
+
+    assert({
+      given: 'no features argument',
+      should: 'render the component',
+      actual: $('.features').length,
+      expected: 1
+    });
+
+    assert({
+      given: 'no features argument',
+      should: 'provide an empty array of features via context',
+      actual: $('.features').text(),
+      expected: ''
+    });
+  }
+
+  {
+    const { assert } = should();
+    const Component = createTestComponent();
+    const features = [];
+    const Page = withFeatureToggles({ features })(Component);
+
+    const $ = dom.load(render(<Page />));
+
+    assert({
```

Comment:
we don't need to test that the component exists, the `.text()` tests tell us the component exists, so we can remove all the `.length` tests.